pos_customer_wallet/models/res_partner.py

```diff
@@ -66,4 +89,5 @@ def get_wallet_balance_all(self, all_partner_ids, all_account_ids):
         res = super().get_wallet_balance_all(all_partner_ids, all_account_ids)
         res.append(self.get_wallet_balance_bank_statement_line(all_partner_ids))
```

Comment:
if we add pos_payment, we should remove bank_statement_line ? or did i missed something ?

Reply:
Forget it, I did'nt saw commit after this one.